types/module/module.go

```diff
@@ -163,17 +161,21 @@ func (bm BasicManager) RegisterGRPCGatewayRoutes(clientCtx client.Context, rtr *
 // AddTxCommands adds all tx commands to the rootTxCmd.
 func (bm BasicManager) AddTxCommands(rootTxCmd *cobra.Command) {
 	for _, b := range bm {
-		if cmd := b.GetTxCmd(); cmd != nil {
-			rootTxCmd.AddCommand(cmd)
+		if mod, ok := b.(interface {
+			GetTxCmd() *cobra.Command
```

Comment:
For reference, this replicates the behavior from the Core API Adapter: https://github.com/cosmos/cosmos-sdk/blob/adb03ecc59a24d276a2bfb83347009ad770192f2/types/module/core_module.go#L137:145